utils/config/daos.yml

```diff
@@ -13,7 +13,7 @@
 # Name must match name specifed in the daos_server.yml file on the server.
 #
 # default: daos
-name: daos
+#name: daos
```

Comment:
I think this has to stay uncommented - the server name is required, I don't think there is a default.

Reply:
There's a default value in the new file:  src/control/client/config.go that is part of pull 430.  I can uncomment this line for now anyway.  It will not hurt anything at this time.  That said, I see that the built in default for 'SystemName' is "daos_server" and not "daos", so I have to update the built in default as part of src/control/client/config.go to match.

Reply:
For the moment, I pushed a new daos.yml that shows the updated server name default, but leaves the entry commented out.  If you still want it uncommented, I will make that change.  Just let me know.

Reply:
The default name in daos_server.yml is "daos" - we should be consistent with the defaults between the two. I'd lean toward changing the code and this file to use "daos" as a default, since this is the DAOS system (cluster) name not some individual server. @johannlombardi should be the decider on this, though.

Leaving it commented is fine, looks like that's what daos_server.yml does as well. I was confused in my earlier comment. :)